http/http.go

```diff
@@ -163,7 +163,7 @@ func SetupRoutes(handle *graph.Handle, cfg *config.Config) {
 func Serve(handle *graph.Handle, cfg *config.Config) {
 	SetupRoutes(handle, cfg)
 	glog.Infof("Cayley now listening on %s:%s\n", cfg.ListenHost, cfg.ListenPort)
-	fmt.Printf("Cayley now listening on %s:%s\n", cfg.ListenHost, cfg.ListenPort)
+	fmt.Printf("Cayley now listening on http://%s:%s\n", cfg.ListenHost, cfg.ListenPort)
```

Comment:
Just amend the `glog.Infof` line? Why have you added a new line repeating almost the exact same thing?